packages/components/pie-textarea/README.md

```diff
@@ -29,70 +20,20 @@ This component can be easily integrated into various frontend frameworks and cus
 To install `pie-textarea` in your application, run the following on your command line:
 
 ```bash
-# npm
-$ npm i @justeattakeaway/pie-textarea
+npm i @justeattakeaway/pie-textarea
```

Comment:
Should this be pointing to the pie-webc package instead?

Reply:
Actually ignore this the other packages seem to follow the same. Unsure why tho.